apps/android_rpc/tests/android_rpc_test.py

```diff
@@ -21,59 +21,86 @@
 arch = "arm64"
 target = "llvm -target=%s-linux-android" % arch
 
+test_opencl = True
+test_vulkan = False
```

Comment:
Consider making these required command line options so we do not have broken defaults depending on the user's configuration (e.g., a CPU-only android device). We should also update the Android install [docs](https://github.com/dmlc/tvm/tree/master/apps/android_rpc) to describe the test options (CPU, OpenCL, Vulkan).

Reply:
Okay, will work on comment